language/move-vm/runtime/src/runtime.rs

```diff
@@ -254,6 +254,42 @@ impl VMRuntime {
         )
     }
 
+    pub(crate) fn execute_script_function(
+        &self,
+        module: &ModuleId,
+        function_name: &IdentStr,
+        ty_args: Vec<TypeTag>,
+        args: Vec<Vec<u8>>,
+        senders: Vec<AccountAddress>,
+        data_store: &mut impl DataStore,
+        cost_strategy: &mut CostStrategy,
+        log_context: &impl LogContext,
+    ) -> VMResult<()> {
+        let (func, ty_args, params) = self.loader.load_function(
+            function_name,
+            module,
+            &ty_args,
+            true,
```

Comment:
Nit: store in a descriptively named variable or add a comment saying what this means?